pkg/base/write.go

```diff
@@ -48,6 +48,43 @@ func (b *Base) WriteBase(options WriteOptions) error {
 	return nil
 }
 
+func (b *Base) WriteKotsKinds(options WriteOptions) error {
```

Comment:
this should not be a method of `Base`.

Reply:
I meant to clean that up before submitting the final version of this PR, sorry!